images/openvswitch/Dockerfile

```diff
@@ -4,18 +4,13 @@
 # The standard name for this image is openshift/openvswitch
 #
 
-FROM centos:centos7
+FROM openshift/origin-base
 
 MAINTAINER Scott Dodson <sdodson@redhat.com>
 
 ADD https://copr.fedoraproject.org/coprs/maxamillion/origin-next/repo/epel-7/maxamillion-origin-next-epel-7.repo /etc/yum.repos.d/
 
-# TODO: systemd update from centos 7.1 -> 7.2 is broken, remove this once 7.2
-# base images land
-RUN yum swap -y -- remove systemd-container\* -- install systemd systemd-libs
```

Comment:
Looks like this is safe to remove now because centos:centos7 is now centos 7.2 based. woot